Gems/Atom/Tools/AtomToolsFramework/Code/Source/Util/Util.cpp

```diff
@@ -400,4 +401,27 @@ namespace AtomToolsFramework
 
         return paths;
     }
+
+    AZStd::set<AZStd::string> GetPathsInSourceFoldersMatchingWildcard(const AZStd::string& wildcard)
+    {
+        AZStd::set<AZStd::string> results;
```

Comment:
Is ordered traversal needed for this or is an `unordered_set` sufficient?

Reply:
I started with unordered, which is probably sufficient and optimal. I kept needing to see the sorted data for visual comparison. So I changed it while iterating. But I can change it back and follow up.